### ISSUE TITLE
Change Text color in SegmentedControlWidget on selection.

### ISSUE THREAD

**Issue by Yash-Khattar** [Clueless-Community/Spectrum-UI#163]
https://github.com/Clueless-Community/Spectrum-UI/assets/67179751/bf445357-31e6-4a01-8b90-35dee79a86f8

In light mode, 
the color of the text in SegmentedControlWidget should change to white in order to be clearly visible on selection 
like the RadioChips.

I'm planning to do something like this, 

https://github.com/Clueless-Community/Spectrum-UI/assets/67179751/0d2eec71-c217-46b9-8a4c-f206e26886a1

I would like to solve this issue, Please assign it to me as a part of GSSOC'23

**Comment by Saksham0205** [comment]
can you please assign me this issue under GSSoc 2023. I really want to work on this issue.

**Comment by Subhash23jan** [comment]
I know this issue,please give me a chance to resolve this

**Comment by ombhojane** [comment]
Hey, I can work on this, please assign this issue to me
GSSOC 2023 :)

**Comment by Apreksha** [comment]
Hey, I'm a GSSoCC'23 contributor. I would like to change the text color in segmented control widget on selection. Could you please assign this task to me.

**Comment by Khushal747** [comment]
please assign me this issue under GSSOC'23

**Comment by Yash-Khattar** [comment]
@Debajyoti14 
Hey, sorry for tagging, but can you please review the issue.
Thank you.

**Comment by Debajyoti14** [comment]
@Yash-Khattar assigning you

**Comment by Debajyoti14** [comment]
@Yash-Khattar Any Progress?

**Comment by Yash-Khattar** [comment]
@Debajyoti14 Yeah I've made the required changes in the code, will submit the PR by today.
